packages/analytics/metric-cards/src/index.ts

```diff
@@ -0,0 +1,16 @@
+import type { App } from 'vue'
+import MetricCards from './components/MetricCards.vue'
+
+// Export Vue plugin as the default
+export default {
+  // Customize Vue plugin options as desired
+  // Providing a `name` property allows for customizing the registered
+  // name of your component (useful if exporting a single component).
+  install: (app: App, options: { name?: string, [key: string]: any } = {}): void => {
+    app.component(options.name || 'MetricCards', MetricCards)
+  },
+}
```

Comment:
Remove this plugin export; we would _always_ want to import these where needed for proper tree-shaking.